src/ripple/protocol/impl/STArray.cpp

```diff
@@ -32,13 +32,13 @@ STArray::STArray()
     //v_.reserve(reserveSize);
 }
 
-STArray::STArray (STArray&& other)
+STArray::STArray (STArray&& other) noexcept
     : STBase(other.getFName())
     , v_(std::move(other.v_))
 {
 }
 
-STArray& STArray::operator= (STArray&& other)
+STArray& STArray::operator= (STArray&& other) noexcept
 {
     setFName(other.getFName());
```

Comment:
`noexcept(setFName(other.getFName()))` evaluates to false, but I believe it could be made true.

Reply:
Good catch. Done in 68c3ec29f